src/components/CompareResults/ResultsTable.tsx

```diff
@@ -112,8 +112,9 @@ const cellsConfiguration: CompareResultsTableConfig = [
 
     gridWidth: '1fr',
   },
-  { key: 'buttons', gridWidth: '2fr' },
-  { key: 'expand', gridWidth: '0.2fr' },
+  // We use the real pixel value for the buttons, so that everything is better aligned.
+  { key: 'buttons', gridWidth: `calc(3.5 * 34px)` }, // 2 or 3 buttons, so at least 3*34px, but give more so that it can "breathe"
```

Comment:
I wasn't sure if we should add some more space, but I find that value doesn't look too bad.

It's especially useful when the "tags" are numerous on the row above, so that the sight can catch the run values easily (see the screenshots for the "main page").